package/python-kiwi-spec-template

```diff
@@ -127,6 +127,7 @@ Provides:       kiwi-image-wsl-requires = %{version}-%{release}
 Obsoletes:      kiwi-image-wsl-requires < %{version}-%{release}
 %if "%{_vendor}" != "debbuild"
 Provides:       kiwi-image:docker
+Provides:       kiwi-image:appx
 Provides:       kiwi-image:wsl
```

Comment:
I'd say this one needs to go away, isn't it? there is no `wsl` image supported in our schema

Reply:
makes sense

Reply:
done